administrator/components/com_guidedtours/services/provider.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * @package       Joomla.Administrator
+ * @subpackage    com_guidedtours
+ *
+ * @copyright     (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
```

Comment:
```suggestion
 * @copyright     (C) 2023 Open Source Matters, Inc. <https://www.joomla.org>
```

and in all other files